Client/TabManagement/Tab.swift

```diff
@@ -660,7 +660,10 @@ class Tab: NSObject {
     @objc
     func reloadPage() {
         reload()
-        self.webView?.scrollView.refreshControl?.endRefreshing()
+
+        DispatchQueue.main.asyncAfter(deadline: .now() + .seconds(1)) {
```

Comment:
Heya. So, I was thinking about this and, just having a delay seems.... it doesn't feel good. Is there something we can do to, for example, maybe observe the webView's loading progress, and call this when it's done loading?